server/routes/index.js

```diff
@@ -1,18 +1,28 @@
+require('dotenv').config();
 const express = require('express');
 const router = express.Router();
+const auth = require('../Middleware/auth');
+
 
 const { createPizza , getPizzas} = require('../controllers/pizzaController');
 const { createOrder, getOrders, getOrderDetails } = require('../controllers/orderItemController');
 // const { getItems } = require('../controllers/orderController');
 
+const { registerUser , loginUser } = require('../controllers/userController')
+
+
+router.post('/register', registerUser)
+router.post('/login', loginUser)
 
-router.post('/create/pizza',createPizza)
-router.get('/menu/pizzas', getPizzas);
+router.post('/create/pizza', auth, createPizza)

```

Comment:
Will there be a difference between creating a pizza and ordering a pizza? Or shall everyone ordering a pizza be allowed to also create?

Reply:
I will tweak the authorization to be only on the pizza creation and order retrieval such that in the future only selected users/resturant owners  are allowed to perform such.